ext/ldap/ldap.c

```diff
@@ -591,6 +600,24 @@ static int _php_ldap_control_from_array(LDAP *ld, LDAPControl** ctrl, zval* arra
 	if (tmpstring != NULL) {
 		zend_string_release(tmpstring);
 	}
+	if (tmpstrings1 != NULL) {
+		int i;
+		for (i = 0; i < num_tmpstrings1; ++i) {
+			if (tmpstrings1[i] != NULL) {
```

Comment:
I don't think it's possible for the strings to be null here?

Reply:
fixed in new commit